net-vpn/candy/metadata.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>yangmame@icloud.com</email>
+		<name>Xin Yang</name>
+	</maintainer>
+		<maintainer type="project" proxied="proxy">
```

Comment:
please keep this line aligned with above

also I can help to co-maintain this package, so you can add my metadata here